src/vs/workbench/contrib/debug/common/debug.ts

```diff
@@ -749,12 +749,12 @@ export interface IDebugAdapterNamedPipeServer {
 	readonly path: string;
 }
 
-export interface IDebugAdapterInlineImpl extends IDisposable {
+interface IDebugAdapterInlineImpl extends IDisposable {
 	readonly onDidSendMessage: Event<DebugProtocol.Message>;
 	handleMessage(message: DebugProtocol.Message): void;
 }
 
-export interface IDebugAdapterImpl {
+interface IDebugAdapterImpl {
```

Comment:
Seems like this and `IDebugAdapterInlineImpl` should be exported since they are part of `IAdapterDescriptor` which is exported

Reply:
And `IDebugAdapterExecutableOptions`

Reply:
Indeed, the `ts-unused-exports` tool could be smarter and omit "indirect" exports...

Reply:
I've reverted the mentioned symbols.